game/src/app.rs

```diff
@@ -387,9 +387,9 @@ impl App {
                     let road = draw_map.get_r(id);
                     for lane in &road.lanes {
                         agents_on.push(Traversable::Lane(lane.id));
-                        for bs in &map.get_l(lane.id).bus_stops {
-                            if show_objs.show(&ID::BusStop(*bs)) {
-                                bus_stops.push(draw_map.get_bs(*bs));
+                        for ts in &map.get_l(lane.id).transit_stops {
```

Comment:
Example boilerplate to rename "bus stop" to "transit stop". You can pretty much ignore this commit